jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -113,7 +113,7 @@ def __init__(self, *args, **kwargs):
 
         self._cached_properties = ['image_layers']
 
-        if self.config not in ['specviz', 'cubeviz']:
+        if self.config not in ['specviz', 'specviz2d', 'cubeviz']:
             # TODO [specviz2d, mosviz] x_display_unit is not implemented in glue for image viewer
             # used by spectrum-2d-viewer
```

Comment:
is this still relevant?

Reply:
For the MosvizImageView, I will make sure to update the comment to reflect this!